### ISSUE TITLE
[Enhancement]: Display Chapter Art while playing.

### ISSUE THREAD

**Issue by Majester** [advplyr/audiobookshelf#2660]
### Describe the feature/enhancement

Display art for each chapter. This has two uses.
1. Some podcasts have individual chapter art that changes on your display as you reach that timestamp. This is particularly useful when the podcaster is referring to an image. While podcasting isn't a visual medium by default, this is useful and quite nice on car displays - my Android Auto supports it.
2. Several of my children's books have chapter illustrations that i would like to display as each Chapter begins. Some even have art mid-chapter which could display as the story progress. The art is effectively tied to a timestamp. I am currently manually pulling the art from the epub. The kids find it very engaging.

**Comment by advplyr** [comment]
There is a feature request open for showing cover art per podcast episode. You are requesting to also support cover art per chapter in a podcast episode, as well as cover art per chapter of a book?

**Comment by Majester** [comment]
Correct. It thus allows for illustrations to be displayed at specified time-codes. There is currently a working solution in the podcast namespace ( https://podcastnamespace.org/) as an extension of podcast Chapters, if you think of a chapters as pre-made bookmarks.
